### PR TITLE
Add frontend & treeherder links in the summary comment

### LINKED ISSUE #477: Link issues page from our frontend in the summary comment
To avoid "losing" issues in Phabricator when a new build is triggered.

### PR DESCRIPTION
**Pull request by La0**
Fixes #477

### CODE REVIEW

File: tools/code_review_tools/treeherder.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 import slugid
 from thclient import TreeherderClient
 
+JOBS_URL = "https://treeherder.mozilla.org/#/jobs"
+
 
 def get_job_url(task_id, run_id, **params):
```
Comment (marco-c) [inline tools/code_review_tools/treeherder.py]: I've just realized that 1be0fb71b98cf7dfe659dcef33b36dda3075c818 was not fully correct, as the URL for task failures don't contain the repo name.

Maybe we should make repository and revision required parameters like for `get_revision_url`

File: bot/code_review_bot/report/base.py [hunk 2]
```diff
@@ -121,7 +124,13 @@ def stats(analyzer, items):
...
 
     def build_comment(
-        self, revision, issues, bug_report_url, patches=[], task_failures=[]
+        self,
+        revision,
+        issues,
+        bug_report_url,
+        patches=[],
+        task_failures=[],
+        frontend_url=None,
```
Comment (marco-c) [inline bot/code_review_bot/report/base.py]: Nit: Can we make this a required argument, so we can avoid checking it it is set

File: bot/code_review_bot/report/base.py [hunk 3]
```diff
@@ -44,6 +44,9 @@
 The analysis task [{name}]({url}) failed, but we could not detect any issue.
 Please check this task manually.
 """
+FRONTEND_LINKS = """
+You can view these defects on [the code-review frontend]({frontend_url}) and on [Treeherder]({treeherder_url}).
```
Comment (marco-c) [inline bot/code_review_bot/report/base.py]: Nit: any idea for a better name than `code-review frontend`?

File: tools/code_review_tools/treeherder.py (hunk 1 above)
Comment (La0) [reply tools/code_review_tools/treeherder.py]: You mean `get_job_url` ?

Comment (marco-c) [reply tools/code_review_tools/treeherder.py]: I mean, maybe repository and revision should be required for get_job_url like they are for get_revision_url

File: bot/code_review_bot/report/base.py (hunk 3 above)
Comment (La0) [reply bot/code_review_bot/report/base.py]: Nope. Just use the domain name `code-review.moz.tools` ?

File: tools/code_review_tools/treeherder.py (hunk 1 above)
Comment (La0) [reply tools/code_review_tools/treeherder.py]: I'll make a single function then

File: tools/code_review_tools/treeherder.py [hunk 4]
```diff
@@ -5,15 +5,23 @@
...
-def get_job_url(task_id, run_id, **params):
+
+def get_job_url(repository, revision, task_id=None, run_id=None, **params):
     """Build a Treeherder job url for a given Taskcluster task"""
-    treeherder_client = TreeherderClient()
-    uuid = slugid.decode(task_id)
+    assert isinstance(repository, str) and repository, "Missing repository"
+    assert isinstance(revision, str) and revision, "Missing revision"
+
+    params.update({"repo": repository, "revision": revision})
```
Comment (marco-c) [inline tools/code_review_tools/treeherder.py]: Nit: assert that `repo` and `revision` are not in params

File: bot/code_review_bot/report/base.py (hunk 3 above)
Comment (marco-c) [reply bot/code_review_bot/report/base.py]: Let's keep `code-review frontend` then

File: tools/code_review_tools/treeherder.py (hunk 4 above)
Comment (La0) [reply tools/code_review_tools/treeherder.py]: Done

**Pull request merged**